build/docker/Dockerfile

```diff
@@ -31,3 +32,22 @@ RUN cd /usr/src/gtest && cmake . && make && cp libgtest.a /usr/lib
 RUN git clone https://github.com/google/gtest-parallel.git
 
 ENV JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
+
+## Website build dependencies
+
+# Install Ruby-2.4.1
+RUN apt-get install -y
+RUN gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB && \
```

Comment:
I followed the instruction son Ruby page on how to install on Ubuntu 16.04. 
I don't think this should be an issue since the release was signed with that key and that won't change. The key is verified automatically in the `rvm` tool. 

In any case this won't affect our build, since this only happens when re-creating the Pulsar build image, not when actually doing our CI job.